policy-controller/src/main.rs

```diff
@@ -128,6 +128,8 @@ async fn main() -> Result<()> {
     let mut prom = <Registry>::default();
     let status_metrics =
         status::ControllerMetrics::register(prom.sub_registry_with_prefix("resource_status"));
+    let status_index_metrcs =
+        status::IndexMetrics::register(prom.sub_registry_with_prefix("resource_status"));
```

Comment:
if these are going to share a prefix, they should just be given the same subregistry (otherwise we'll actually create two separate data structures internally).